### ISSUE TITLE
Importing browser data on iOS

### ISSUE THREAD

**Issue by nbhasin2** [mozilla/platform-tilt#6]
Browsing information like history, bookmarked sites, and cookies isn’t accessible to third-party browsers on iOS, and there’s no API to allow this data to be imported. While this is sensitive data, similar import functionality is possible on all major desktop platforms, and iOS is able to mediate access to other sensitive data with user consent (e.g., [health and fitness data](https://support.apple.com/guide/iphone/share-your-health-data-iph5ede58c3d/ios)). Not being able to import data creates significant friction to change from Safari - a user should be allowed to bring their data with them to another browser.

**Comment by willfs84** [comment]
Yes and crucially, 3rd party browsers should be able to do this on first run (when a user first installs and opens their browser). For example, Firefox on Windows does this when you open it for the first time.

![Image](https://github.com/user-attachments/assets/ed3dca3a-5a9b-496f-8659-938f1547e25e)

**Comment by willfs84** [comment]
Apple claims this is possible as of 18.2 but no browser has been able to implement it? https://support.apple.com/en-ca/guide/iphone/iph1852764a6/18.0/ios/18.0

**Comment by brandoncote** [comment]
Orion is able to, but Apple requires the user to start the process in Settings app
